pom.xml

```diff
@@ -212,6 +212,26 @@
 				<artifactId>org.eclipse.swt</artifactId>
 				<version>${version.swt}</version>
 			</dependency>
+			<dependency>
+				<groupId>org.eclipse.platform</groupId>
```

Comment:
Having different enabled platforms here make SWT unable to find the appropriate files. I comment locally linux & windows to make it work on mac.

Reply:
I'm not sure if I understand correctly, so does it works with your edit to have this profiles? Or does it not work at all? Its a bit hard to test for me as I don't have a mac at hand...

Reply:
When omitting <name>Linux</name> in the swt-unix profile, then a Mac will load profile swt-mac and swt-linux, because Mac is also Unix. This makes both the unix linux and macos libraries appear in the classpath which prevent SWT to work on mac.

Adding `<name>Linux</name>` in the swt-linux section avoid loading this lib for Mac and thus fix mac apps.

Reply:
I have added the name now to tohe profile